### PR TITLE
Allow flux scripts in query string for DE

### LINKED ISSUE #50: As a programmer I can have a design for host viewer in the nav bar

### PR DESCRIPTION
**Pull request by bthesorceror**
Closes #50

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/data_explorer/containers/DataExplorer.tsx [hunk 1]
```diff
@@ -148,55 +149,32 @@ export class DataExplorer extends PureComponent<Props, State> {
...
-      loadDE,
-      timeRange,
-      dashboards,
-      autoRefresh,
-      queryDrafts,
-      handleGetDashboards,
-    } = this.props
-    const {query} = qs.parse(location.search, {ignoreQueryPrefix: true})
+    const {loadDE, timeRange, autoRefresh, queryDrafts} = this.props
+    const {query, script} = this.queryString
```
Comment (OfTheDelmer) [inline ui/src/data_explorer/containers/DataExplorer.tsx]: I somewhat expected `queryString` to a string

File: ui/src/data_explorer/containers/DataExplorer.tsx [hunk 2]
```diff
@@ -304,6 +260,57 @@ export class DataExplorer extends PureComponent<Props, State> {
...
+
+  private get queryString(): {query?: string; script?: string} {
+    return qs.parse(location.search, {ignoreQueryPrefix: true})
+  }
+
+  private get service(): Service {
+    const {services, sourceLink} = this.props
+    let service: Service = null
+
+    if (sourceLink.indexOf('services') !== -1) {
```
Comment (OfTheDelmer) [inline ui/src/data_explorer/containers/DataExplorer.tsx]: we could use `sourceLink.includes`

Comment (bthesorceror) [reply ui/src/data_explorer/containers/DataExplorer.tsx]: fair, that was copy pasta

File: ui/src/data_explorer/containers/DataExplorer.tsx (hunk 1 above)
Comment (bthesorceror) [reply ui/src/data_explorer/containers/DataExplorer.tsx]: its a parsed query string

**Pull request merged**